### PR TITLE
Fixed DateSorting

### PR DESCRIPTION
**Pull request by XWB**

### CODE REVIEW

**Comment by stof** [comment]
@XWB when creating PRs, please use your personal fork to host the feature branch rather than the FOS repo. This has several advantages:

- it avoids creating versions on Packagist for these temporary branches
- it avoids triggering a second Travis build for the branch itself (the Travis rate limiting is per repo owner, so this affects the whole FOS organization, even though we don't have so much activity that it hurts us, but it does for some organizations)
- it avoids sending Travis notification emails to the FOS mailing-list for each build of your PR (PR builds don't trigger notifications, but branch builds do)

File: Sorting/DateSorting.php [hunk 1]
```diff
@@ -29,10 +29,10 @@ class DateSorting extends AbstractOrderSorting
      */
     protected function compare(CommentInterface $a, CommentInterface $b)
     {
-        if ($a->getCreatedAt() === $b->getCreatedAt()) {
+        if ($a->getCreatedAt()->getTimestamp() === $b->getCreatedAt()->getTimestamp()) {
             return 0;
         }
 
-        return $a->getCreatedAt() < $b->getCreatedAt() ? -1 : 1;
+        return $a->getCreatedAt()->getTimestamp() < $b->getCreatedAt()->getTimestamp() ? -1 : 1;
```
Comment (stof) [inline Sorting/DateSorting.php]: this is not needed here. This comparison operator was working fine. PHP is able to compare DateTime objects

**Comment by XWB** [comment]
@stof You are right, sorry for that. I wasn't aware that I was pushing to the FOS repository.

**Comment by stof** [comment]
@XWB btw, it might be worth adding a test for that to avoid regressions

**Comment by XWB** [comment]
@stof See https://github.com/FriendsOfSymfony/FOSCommentBundle/pull/472

**Pull request merged**